Cart-Page/cart.css

```diff
@@ -0,0 +1,354 @@
+}
+
+/*****Colors*****/
+
+:root {
+  --primary-color: #3db5fa;
+  --primary-bg-color: rgb(24, 22, 22);
+  --secondary-bg-color: rgb(34, 32, 32);
+  --white-color: white;
+  --black-color: black;
+  --font-size-XX-large: clamp(3rem, 12vw, 10rem);
+  --font-size-X-large: clamp(1.2rem, 2.5vw, 10rem);
+  --font-size-large: clamp(1rem, 1.5vw, 10rem);
+  --font-size-regular: clamp(12px, 2.5vw, 1rem);
+  --font-size-small: clamp(8px, 1vw, 1rem);
+}
+body {
+  font-family: "Montserrat", sans-serif;
+  display: grid;
+  grid-template-rows: 2.5rem minmax(calc(100vh - 7.5rem), 1fr) 5rem;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
+}
+
+/*****Navigation Bar For Desktop *****/
+
+.nav-bar {
+  position: fixed;
+  z-index: 1;
+  width: 100%;
+  grid-area: header;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 8px;
+  background-color: var(--primary-bg-color);
+  border-bottom: 2px solid var(--primary-color);
+}
+.nav-bar__img--logo {
+  margin-left: 1rem;
+  min-width: 6rem;
+  max-width: 10rem;
+}
+.nav-bar__list {
+  margin-right: 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.nav-bar__item {
+  color: var(--primary-color);
+  font-size: var(--font-size-medium);
+  padding: 5px 8px;
+  border: 2px solid transparent;
+}
+.nav-bar__img--icon {
+  min-width: 1rem;
+  max-width: 1.5rem;
+}
+.nav-bar__item:hover {
+  border-bottom: 2px solid var(--primary-color);
+}
+
+/* Navigation Bar For Tab and Mobile */
+
+.nav-bar--hamburger {
+  display: none;
+}
+.nav-bar__img--hamburger {
+  width: 1.5rem;
+}
+
+/* Hamburger List */
+
+.hamburger {
+  grid-area: header;
+  visibility: hidden;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  z-index: 5;
+  position: fixed;
+  overflow-y: scroll;
+  top: 0;
+  width: 0;
+  transition: width 0.5s;
+  border-right: 2px solid var(--primary-color);
+  background-color: var(--primary-bg-color);
+}
+.hamburger__header--wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+.hamburger__title {
+  font-size: var(--font-size-X-large);
+  margin: 1rem;
+  color: var(--primary-color);
+}
+.hamburger__list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.hamburger__item {
+  margin: 1rem;
+  font-size: var(--font-size-X-large);
+  color: var(--white-color);
+}
+.hamburger__close-btn {
+  position: absolute;
+  right: 0;
+  top: 4px;
+  margin: 1rem;
+  width: 1rem;
+}
+.hamburger__item:hover {
+  color: var(--primary-color);
+  background-color: var(--secondary-bg-color);
+}
+.hamburger__btn {
+  width: 85%;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  padding: 8px;
+  font-size: var(--font-size-large);
+  background-color: var(--primary-bg-color);
+  border: 2px solid var(--primary-color);
+}
+
+/*** Cart badge ***/
+.badge__wrapper {
+  display: inline-block;
+  position: relative;
+}
+.badge--logo {
+  font-size: 70%;
+  font-weight: 900;
+  background-color: red;
```

Comment:
can use variable name

Reply:
This is only for badge background color, Should I convert it into a variable?